controllers/front/ajax.php

```diff
@@ -33,90 +33,93 @@ public function postProcess()
+        $total_including_tax = $orderTotalWithFee->toPrecision(2);
+        $total_excluding_tax = $orderTotalNoTaxWithFee->toPrecision(2);
 
-                $taxConfiguration = new TaxConfiguration();
-                $presentedCart = $this->cart_presenter->present($this->context->cart);
+        $taxConfiguration = new TaxConfiguration();
+        $presentedCart = $this->cart_presenter->present($this->context->cart);
 
-                $presentedCart['totals'] = [
-                    'total' => [
-                        'type' => 'total',
-                        'label' => $this->translator->trans('Total', [], 'Shop.Theme.Checkout'),
-                        'amount' => $taxConfiguration->includeTaxes() ? $total_including_tax : $total_excluding_tax,
-                        'value' => $locale->formatPrice(
-                            $taxConfiguration->includeTaxes() ? $total_including_tax : $total_excluding_tax, $currency->iso_code),
-                    ],
-                    'total_including_tax' => [
-                        'type' => 'total',
-                        'label' => $this->translator->trans('Total (tax incl.)', [], 'Shop.Theme.Checkout'),
-                        'amount' => $total_including_tax,
-                        'value' => $locale->formatPrice($total_including_tax, $currency->iso_code),
-                    ],
-                    'total_excluding_tax' => [
-                        'type' => 'total',
-                        'label' => $this->translator->trans('Total (tax excl.)', [], 'Shop.Theme.Checkout'),
-                        'amount' => $total_excluding_tax,
-                        'value' => $locale->formatPrice($total_excluding_tax, $currency->iso_code),
-                    ],
-                ];
+        $presentedCart['totals'] = [
+            'total' => [
+                'type' => 'total',
+                'label' => $this->translator->trans('Total', [], 'Shop.Theme.Checkout'),
+                'amount' => $taxConfiguration->includeTaxes() ? $total_including_tax : $total_excluding_tax,
+                'value' => $locale->formatPrice(
+                    $taxConfiguration->includeTaxes() ? $total_including_tax : $total_excluding_tax, $currency->iso_code),
+            ],
+            'total_including_tax' => [
+                'type' => 'total',
+                'label' => $this->translator->trans('Total (tax incl.)', [], 'Shop.Theme.Checkout'),
+                'amount' => $total_including_tax,
+                'value' => $locale->formatPrice($total_including_tax, $currency->iso_code),
+            ],
+            'total_excluding_tax' => [
+                'type' => 'total',
+                'label' => $this->translator->trans('Total (tax excl.)', [], 'Shop.Theme.Checkout'),
+                'amount' => $total_excluding_tax,
+                'value' => $locale->formatPrice($total_excluding_tax, $currency->iso_code),
+            ],
+        ];
```

Comment:
from what it looks to me we somehow override some prestashop variables, I am not sure if we want to do this, what happens if someone else decides to do the same?